src/linker/Linker.Dataflow/ReflectionMethodBodyScanner.cs

```diff
@@ -926,6 +948,28 @@ public override bool HandleCall (MethodBody callingMethodBody, MethodReference c
 						reflectionContext.RecordUnrecognizedPattern ($"Activator call '{reflectionContext.MethodCalled.FullName}' inside '{reflectionContext.MethodCalling.FullName}' is not yet supported");
 						break;
 
+					//
+					// System.Runtime.CompilerServices.RuntimeHelpers
+					//
+					// RunClassConstructor (RuntimeTypeHandle type)
+					//
+					case IntrinsicId.RuntimeHelpers_RunClassConstructor: {
+							reflectionContext.AnalyzingPattern ();
+							foreach (var TypeHandleValue in methodParams [0].UniqueValues ()) {
```

Comment:
Local variable should start with lower case `TypeHandleValue` -> `typeHandleValue`.